app/models/concerns/waste_carriers_engine/can_have_registration_attributes.rb

```diff
@@ -109,14 +101,10 @@ def company_no_required?
       end
 
       def main_people
-        return [] unless key_people.present?
```

Comment:
I need some context as to why this change has been made. Was it just housekeeping (deleting unnecessary code), or something related to the PR I don't understand???

I've spent some time invested in the Rails console both pre and post change trying to see for myself, and I can see it definitely makes a change to what gets returned when no `key_people` are present, but I don't think in any way that is meaningful to us.

So I guess I'd just like to understand better how it fits into the PR.

Reply:
Just me opening the class and thinking we don't need that as `has_many` association should return an empty array even after a where is applied even if empty. Is this not the case for MongoId?

Reply:
Sort of. With that line removed it always returns a [Mongoid::Criteria](https://www.rubydoc.info/github/mongoid/mongoid/Mongoid/Criteria) which you then can interact with like an array. So for example if `key_people` does not exist `my_reg.main_people.length` returns 0 whether the line is there or not.

Which is why I said it's probably fine and not going to break anything, but perhaps in future use commits to separate out what is the feature, and what is other bits like housekeeping, fixes, code formatting changes etc. I just wasn't sure if the single document stuff relied on this change because of something I wasn't seeing.

Reply:
I understand! Let me scrap this change out of this PR :)